--- a/Makefile	2016-11-22 22:50:31.000000000 +0100
+++ b/Makefile	2016-12-03 20:16:57.000000000 +0100
@@ -144,9 +144,12 @@
 	##
 	TAR	       = gtar
 	LD	       = ld
+	CFLAGS        += -I/usr/local/include
 	CFLAGS        += -I/usr/local/opt/libiconv/include
+	CFLAGS        += -mmacosx-version-min=10.7
+	LDLIBS        += -L/usr/local/lib
 	LDLIBS        += -L/usr/local/opt/libiconv/lib/ -liconv -lstdc++
-	LDFLAGS       += -dylib -arch x86_64 -macosx_version_min 10.10
+	LDFLAGS       += -dylib -arch x86_64 -macosx_version_min 10.7
 	DLEXT          = dylib
 	FULLDLEXT      = ${BIOSIG_VERSION}.dylib
 	SONAME_PREFIX  = -install_name # the last space character is crucial
 
